ext/natives/codegen_cfx_natives.lua

```diff
@@ -1731,13 +1731,21 @@ native 'SET_VEHICLE_WHEEL_X_OFFSET'
 	</summary>
 ]]
 
-native 'OVERRIDE_PLAYER_TALKING'
+native 'SET_PLAYER_TALKING_OVERRIDE'
 	arguments {
 		Player 'player',
 		BOOL 'state'
 	}
 	apiset 'client'
 	returns 'void'
+	doc [[
+	<param name="player">The target player.</param>
```

Comment:
I'm not too sure how much the system will like `<param>` below the summary. :/

Reply:
Native gen is always output summary before params, but I will fix.